pkg/kapis/monitoring/v1alpha3/helper.go

```diff
@@ -337,6 +345,26 @@ func (h handler) makeQueryOptions(r reqParams, lvl monitoring.Level) (q queryOpt
 		}
 		q.namedMetrics = model.PVCMetrics
 
+	case monitoring.LevelIngress:
+		q.identifier = model.IdentifierIngress
+		var du *time.Duration
+		// duration param is used in none Range Query to pass vector's time duration.
+		if r.time != "" {
+			s, err := time.ParseDuration(r.duration)
+			if err == nil {
+				du = &s
+			}
+		}
```

Comment:
Why is the `duration` parameter restricted to instant query only

Reply:
There's no place to set  `duration `in the chart controls and it would be confused for users how to set the value.
The following screenshot shows how the instant query used on the UI
![20210916104038](https://user-images.githubusercontent.com/1720333/133540621-5fd90f88-9a9e-4ee0-96a4-601695f6c4fd.png)

Reply:
So here the `duration` is set instead of `start` and `end` time, which looks different from other monitoring pages